node-operators/.pages

```diff
@@ -0,0 +1,5 @@
+title: Node Operators
+hide: false
+nav:
+  - index.md
+  - "RPC node": 'rpc.md'
```

Comment:
eventually there will be a systemd page, so this should be a section, even if it only has one page in it for now